<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalyzeForm</class>
 <widget class="QWidget" name="AnalyzeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnalyzeForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_title
{
	background-color: rgb(50, 50, 50);

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>33</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_id">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>714</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/png/res/png/close.png</normaloff>:/png/res/png/close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QtMaterialTabs" name="widget_tab" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QChartView" name="page"/>
     <widget class="QChartView" name="page_2"/>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialTabs</class>
   <extends>QWidget</extends>
   <header location="global">qtmaterialtabs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AnalyzeForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
